meson.build

```diff
@@ -341,6 +341,17 @@ if not get_option('mxnet-support').disabled()
   endif
 endif
 
+# ncnn
```

Comment:
Suggestion: pkgconfig 'ncnn.pc'
IMO you can use pkg-config, instead checking the ncnn library (or header) in this block.
```
  'ncnn-support': {
    'target': 'ncnn',
    'project_args': { 'ENABLE_NCNN' : 1 }
  },```

Reply:
AFAIU the code is already finding the ncnn library with pkg-config (via meson's `cxx.find_library`)? The header checking code is a fallback for when pkg-config doesn't exist, which is copied verbatim from the logic of finding other dependencies (i.e. see MXnet or ARMNN).

Reply:
OK, we will check this after ncnn packaging is done later.